k8s-manifests/env.yaml

```diff
@@ -12,12 +12,9 @@ data:
   RERANKER_PORT: "80"
   VECTORDB_HOST: "ai-stack-vectordb"
   VECTORDB_PORT: "8000"
-  STOP_TOKEN: "<|endoftext|>"
```

Comment:
Did you remove these intentionally? here & in .envrc?

Reply:
yes not required with openai client